--- origsrc/mdbtools-0.6pre1/configure.in	2004-06-17 21:18:14.000000000 -0500
+++ src/mdbtools-0.6pre1/configure.in	2006-12-24 23:30:35.324750000 -0600
@@ -110,7 +110,6 @@
 fi
 
 AC_SUBST([OPTDIRS])
-AC_CONFIG_FILES([src/Makefile])
 
 
 ##################################################
@@ -176,6 +175,7 @@
 AC_OUTPUT([
 libmdb.pc
 libmdbsql.pc
+src/Makefile
 src/util/Makefile
 src/extras/Makefile
 Makefile include/Makefile
--- origsrc/mdbtools-0.6pre1/doc/Makefile.am	2004-02-08 21:09:53.000000000 -0600
+++ src/mdbtools-0.6pre1/doc/Makefile.am	2006-12-24 23:30:35.324750000 -0600
@@ -34,18 +34,18 @@
 man:	mdb-tables.1 mdb-ver.1 mdb-export.1 mdb-sql.1
 
 mdb-tables.1: mdb-tables.txt
-	- $(TXT2MAN) -P $(PRODUCT) -t $(PRODUCT) -r $(RELEASE) $(srcdir)/$< > $@
+	- $(TXT2MAN) -P $(PRODUCT) -t $(PRODUCT) -r $(RELEASE) $< > $@
 	
 mdb-ver.1: mdb-ver.txt
-	- $(TXT2MAN) -P $(PRODUCT) -t $(PRODUCT) -r $(RELEASE) $(srcdir)/$< > $@
+	- $(TXT2MAN) -P $(PRODUCT) -t $(PRODUCT) -r $(RELEASE) $< > $@
 
 mdb-export.1: mdb-export.txt
-	- $(TXT2MAN) -P $(PRODUCT) -t $(PRODUCT) -r $(RELEASE) $(srcdir)/$< > $@
+	- $(TXT2MAN) -P $(PRODUCT) -t $(PRODUCT) -r $(RELEASE) $< > $@
 
 mdb-schema.1: mdb-schema.txt
-	- $(TXT2MAN) -P $(PRODUCT) -t $(PRODUCT) -r $(RELEASE) $(srcdir)/$< > $@
+	- $(TXT2MAN) -P $(PRODUCT) -t $(PRODUCT) -r $(RELEASE) $< > $@
 
 mdb-sql.1: mdb-sql.txt
-	- $(TXT2MAN) -P $(PRODUCT) -t $(PRODUCT) -r $(RELEASE) $(srcdir)/$< > $@
+	- $(TXT2MAN) -P $(PRODUCT) -t $(PRODUCT) -r $(RELEASE) $< > $@
 
 
--- origsrc/mdbtools-0.6pre1/src/gmdb2/help/C/Makefile.am	2004-02-08 21:09:53.000000000 -0600
+++ src/mdbtools-0.6pre1/src/gmdb2/help/C/Makefile.am	2006-12-24 23:38:35.340375000 -0600
@@ -5,5 +5,5 @@
 lang = C
 omffile = gmdb-C.omf
 entities = legal.xml
-include xmldocs.make
+include $(top_srcdir)/xmldocs.make
 dist-hook: app-dist-hook
