--- origsrc/mdbtools-0.6pre1/src/libmdb/Makefile.am	2004-03-05 23:13:27.000000000 -0600
+++ src/mdbtools-0.6pre1/src/libmdb/Makefile.am	2006-12-24 23:30:35.621625000 -0600
@@ -1,4 +1,5 @@
 lib_LTLIBRARIES	=	libmdb.la
 libmdb_la_SOURCES=	catalog.c mem.c file.c kkd.c table.c data.c dump.c backend.c money.c sargs.c index.c like.c write.c stats.c map.c props.c worktable.c options.c iconv.c
+libmdb_la_LDFLAGS = -no-undefined
 AM_CPPFLAGS	=	-I$(top_srcdir)/include $(GLIB_CFLAGS)
 LIBS = $(GLIB_LIBS) -lm
--- origsrc/mdbtools-0.6pre1/src/odbc/Makefile.am	2004-04-13 19:25:56.000000000 -0500
+++ src/mdbtools-0.6pre1/src/odbc/Makefile.am	2006-12-24 23:36:51.543500000 -0600
@@ -9,13 +9,11 @@
 
 bin_PROGRAMS	=	unittest 
 lib_LTLIBRARIES	=	libmdbodbc.la
-AM_CPPFLAGS	=	-I ../../include $(GLIB_CFLAGS) $(ODBC_CFLAGS)
+AM_CPPFLAGS	=	-I $(top_srcdir)/include $(GLIB_CFLAGS) $(ODBC_CFLAGS)
 libmdbodbc_la_SOURCES=	odbc.c connectparams.c
-libmdbodbc_la_LIBADD=	$(patsubst %, $(SQLDIR)/%, \
-				$(patsubst %.c, %.lo, $(SQLSOURCES))) \
-				$(patsubst %, $(MDBDIR)/%, \
-				$(patsubst %.c, %.lo, $(MDBSOURCES)))
-LIBS	=	@LEXLIB@ $(GLIB_LIBS) $(ODBC_LIBS)
+libmdbodbc_la_LDFLAGS=	-no-undefined
+libmdbodbc_la_LIBADD=  ../libmdb/libmdb.la ../sql/libmdbsql.la
+LIBS	=	$(GLIB_LIBS) $(ODBC_LIBS)
 unittest_LDADD	=	libmdbodbc.la ../libmdb/libmdb.la ../sql/libmdbsql.la
 
 ## Need blank statement to avoid compiling odbc.c
--- origsrc/mdbtools-0.6pre1/src/sql/Makefile.am	2004-02-08 21:10:19.000000000 -0600
+++ src/mdbtools-0.6pre1/src/sql/Makefile.am	2006-12-24 23:30:35.637250000 -0600
@@ -1,5 +1,7 @@
 lib_LTLIBRARIES	=	libmdbsql.la
 libmdbsql_la_SOURCES=	mdbsql.c parser.y lexer.l 
+libmdbsql_la_LDFLAGS=	-no-undefined
+libmdbsql_la_LIBADD=	../libmdb/libmdb.la
 EXTRA_DIST = main.c
 DISTCLEANFILES = parser.c parser.h lexer.c
 AM_CPPFLAGS	=	-I$(top_srcdir)/include $(GLIB_CFLAGS)
